content/support.rst

```diff
@@ -0,0 +1,212 @@
+Support
+#######
+
+:menu_order: 009
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describes Support details.
+
+For access to any of the Support platforms, please contact the Support team.
```

Comment:
Audience: Anyone new / unfamiliar with the process. 
Also only you / me have access to certain platforms (ie Olark), only a few for Freshdesk.

I will keep it to this type of audience.